examples/spa/src/server/AppController.ts

```diff
@@ -0,0 +1,19 @@
+import { Controller } from '@nestjs/common';
+import { Page } from 'nest-next-spa/server';
+
+@Controller()
+export class AppController {
+  @Page('')
+  public index() {
+    return {
+      title: 'Index Page',
```

Comment:
I'd add an injectable here to showcase that it will always be available including client-side navigations.

Something like `ApiService` that would return mock titles could work